headers/kwc-dialog-icon-header.html

```diff
@@ -0,0 +1,219 @@
+<link rel="import" href="../../polymer/polymer-element.html">
+
+<!--
+`kwc-dialog-icon-header` is a header for dialogs with a big icon at the top, and a dismiss icon.
+
+### Examples
+
+Basic:
+
+```
+<kwc-dialog>
+    <kwc-dialog-icon-header slot="header"></kwc-dialog-icon-header>
+    <div slot="content">content here</div>
+</kwc-dialog>
+```
+
+@demo demo/index.html
+-->
+
+<dom-module id="kwc-dialog-icon-header">
+    <template>
+        <style>
+            :host {
+                --color-light-grey: #f4f5f6;
+                --color-text: #253541;
+                display: block;
+                color: var(--color-text);
+                font-family: 'Bariol', sans-serif;
```

Comment:
The font doesn't seem to be used anywhere in this header. (Or if it is, we're missing the typography import.)